src/frontend/src/lib/services/analytics.services.ts

```diff
@@ -1,27 +1,35 @@
+import { browser } from '$app/environment';
 import { PLAUSIBLE_DOMAIN, PLAUSIBLE_ENABLED } from '$env/plausible.env';
 import type { TrackEventParams } from '$lib/types/analytics';
 import { isNullish, nonNullish } from '@dfinity/utils';
-import Plausible from 'plausible-tracker';
 
-let plausibleTracker: ReturnType<typeof Plausible> | null = null;
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
+let plausibleTracker: typeof import('@plausible-analytics/tracker') | null = null;
 
-// eslint-disable-next-line require-await -- We use this service during initialisation among other async functions, so for consistency we force the asynchronicity here.
 export const initPlausibleAnalytics = async () => {
-	if (!PLAUSIBLE_ENABLED || isNullish(PLAUSIBLE_DOMAIN)) {
+	if (
+		!PLAUSIBLE_ENABLED ||
+		isNullish(PLAUSIBLE_DOMAIN) ||
+		!browser ||
+		nonNullish(plausibleTracker)
```

Comment:
Picky question: is non nullish really equals to being initialized? Those are two differents steps as far as I can tell from the code. Though it probably works in practice.